docs/user-guide/configuration.rst

```diff
@@ -13,4 +13,4 @@ The Puppet importer is configured by editing
 
 The importer supports the settings documented in Pulp's `importer config docs`_.
 
-.. _importer config docs: https://pulp-user-guide.readthedocs.org/en/latest/server.html#importers
+.. _importer config docs: https://pulp.readthedocs.org/en/latest/user-guide/server.html#importers
```

Comment:
Use intersphinx for this instead.  Here is [the config for puppet](https://github.com/pulp/pulp_puppet/blob/9a307875acd9ccd8f15ae8f8327b334d7c7af2af/docs/conf.py#L249). Also here are the [docs on intersphinx](http://sphinx-doc.org/ext/intersphinx.html).